### PR TITLE
fix wrongly filtered paths in loader.py

### PR DESCRIPTION
**Pull request by mragank02**
Line 1768, the small change of "path" to "filtered_path"

When I tried to run the code, the "h5" was deleted and error was shown that the desired "h5" file was not found. I got the follwoing error: 

Loading Line databank
Opening file C:\Users\mraga\Databases\cdsd4000\CDSD4000_4150_5650\cdsd_01768_01770.h5 (format=CDSD 4000, cache=True)
Last Modification time: Mon Feb 20 15:58:05 2023
File C:\Users\mraga\Databases\cdsd4000\CDSD4000_4150_5650\cdsd_01768_01770.h5 deprecated:
Metadata in file C:\Users\mraga\Databases\cdsd4000\CDSD4000_4150_5650\cdsd_01768_01770.h5 dont match expected values. See comparison below:
	Expected	File
Key            Expected         	Got
----------------------------------------
last_modificationMon Feb 20 15:58:05 2023	Mon May 31 07:59:56 2010
----------------------------------------
Deleting it!
Traceback (most recent call last):
  File "C:\Users\mraga\Mragank_tuto_CO2_CDSD.py", line 32, in <module>
    sf.load_databank('CDSD4000_4150_5650', load_energies=True, load_columns='all')
  File "C:\Users\mraga\anaconda3\lib\site-packages\radis\lbl\loader.py", line 1625, in load_databank
    self.df0 = self._load_databank(
  File "C:\Users\mraga\anaconda3\lib\site-packages\radis\lbl\loader.py", line 2354, in _load_databank
    df = load_and_concat(database)
  File "C:\Users\mraga\anaconda3\lib\site-packages\radis\lbl\loader.py", line 2242, in load_and_concat
    df = cdsd2df(
  File "C:\Users\mraga\anaconda3\lib\site-packages\radis\api\cdsdapi.py", line 283, in cdsd2df
    df = parse_hitran_file(fname, columns)
  File "C:\Users\mraga\anaconda3\lib\site-packages\radis\api\tools.py", line 45, in parse_hitran_file
    data = _read_hitran_file(
  File "C:\Users\mraga\anaconda3\lib\site-packages\radis\api\tools.py", line 130, in _read_hitran_file
    return np.fromfile(fname, dtype=dt, count=count)
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\mraga\\Databases\\cdsd4000\\CDSD4000_4150_5650\\cdsd_01768_01770.h5'
[4:40](https://em2cdocs.slack.com/archives/D04LBNUULVA/p1678462840712929)

So basically, it deleted the file by itself and gave me the error that the file was not found!
.
.
.

### Description

This pull request is to address ...

Fixes #<Issue Number>

### CODE REVIEW

**Comment by mragank02** [comment]
Here is my code, that I had used to run into the error.  

'''
from radis import SpectrumFactory, load_spec
import numpy as np

distance_in_cm = 1/200
pressure = 1.01325
wstep = 0.005
cutoff = 0
verbose = 2
warnings = False

wav_min = 4250
wav_max = 4260

sf = SpectrumFactory(wavelength_min=wav_min,
                         wavelength_max=wav_max,
                         pressure=pressure,
                         molecule='CO2',
                         isotope='1,2',
                         wstep=wstep,
                         cutoff=cutoff,
                         verbose=verbose,
                         path_length=distance_in_cm,
                         mole_fraction=1e-2,
                         warnings=warnings,
                         )
sf.load_databank('CDSD4000_4150_5650_all', load_energies=True, load_columns='all')
sf.df0.drop(sf.df0.index[sf.df0['polyu']>60], inplace=True)
sf.df0.drop(sf.df0.index[sf.df0['polyl']>60], inplace=True)


spec = sf.non_eq_spectrum(Trot=2500, Tvib=(2600,2600,2600), mole_fraction=1.5e-1, path_length=1/200) # thermal non-eq, path length is in cm

print(spec.conditions)

# spec.rescale_mole_fraction(0.0067)

# spec.apply_slit(4, 'nm', mode='same')
apply_my_slit(spec)
spec.plot('radiance', wunit='nm')

'''

**Comment by erwanp** [comment]
Thanks, merging
